code/association_scan/TensorQTL/TensorQTL.ipynb

```diff
@@ -1104,13 +1104,15 @@
    "outputs": [],
    "source": [
     "[trans_1]\n",
-    "input: input_files, group_by = len(input_files[0]), group_with = \"input_chroms\"\n",
-    "output: nominal = f'{cwd:a}/{_input[0]:bnn}{\"\" if input_chroms[_index] == 0 else \"_%s\" % input_chroms[_index]}.trans_qtl.pairs.tsv.gz'\n",
     "\n",
     "parameter: batch_size = 10000\n",
     "parameter: pval_threshold = 1.0\n",
     "# Permutation testing is incorrect when the analysis is done by chrom\n",
     "parameter: permutation = False\n",
+    "parameter: pval = 0.0\n",
+    "\n",
+    "input: input_files, group_by = len(input_files[0]), group_with = \"input_chroms\"\n",
+    "output: nominal = f'{cwd:a}/{_input[0]:bnn}{\"\" if input_chroms[_index] == 0 else \"_%s\" % input_chroms[_index]}.trans_qtl.p_{pval}.pairs.tsv.gz'\n",
```

Comment:
I prefer changing this from `trans_qtl.p_{pval}` to `trans_qtl{("_p_%" %s pval) if pval > 0.0 else ""}`